web/vibe/web/rest.d

```diff
@@ -186,14 +190,34 @@
 			string getBody(FooType myFoo);
 		}
 		----
+        
+		Further, how function parameters are named may affect the route:
+
+		$(UL
+			$(LI If the first parameter is named `id`, this is interpreted as a
+				leading route component. For example, `getName(int id)` becomes
+				`/:id/name`.)
+				$(B Note that this style of parameter-based URL routing is
+				different than in many other web frameworks, ) where instead
+				ths example would be routed as `/name:id`.
```

Comment:
Shouldn't this be `/name/:id` ?

Reply:
Yes, and I also misspelled "this" in that same line. Typing too quickly. Thank you for catching, will update.